drivers/can/can_mcux_flexcan.c

```diff
@@ -587,17 +591,17 @@ static void mcux_flexcan_transfer_callback(CAN_Type *base,
 
 	switch (status) {
 	case kStatus_FLEXCAN_UnHandled:
-		__fallthrough;
+		/* fallthrough */
```

Comment:
why are you changing these fallthroughs? is it a rebasing error?

Reply:
yes it was.. fixed now